praw/__init__.py

```diff
@@ -238,6 +238,13 @@ def config_boolean(item):
         self.https_proxy = (obj.get('https_proxy') or
                             os.getenv('https_proxy') or None)
         # We use `get(...) or None` because `get` may return an empty string
+
```

Comment:
There's no need to check if `validate_certs` is present assuming it's added to `praw.ini` (it'll always be present).